msteams-platform/get-started/glossary.md

```diff
@@ -260,11 +259,13 @@ Common terms and definitions used in Microsoft Teams developer documentation.
 | [Thread discussion](../tabs/design/tabs.md#thread-discussion) | A conversation posted on a channel or chat between users. <br>**See also** [Conversation](#c); [Channel](#c) |
 | [Teams](../overview.md) | Microsoft Teams is the ultimate message app for your organization. It's a workspace for real-time collaboration and communication, meetings, file and app sharing. |
 | [Teams AI library](../bots/how-to/teams-conversational-ai/teams-conversation-ai-overview.md) | A Teams-centric interface to GPT-based common language models and user intent engines. You can take on complex and expensive tasks of writing and maintaining conversational bot logic to integrate with Large Language Models (LLMs).|
+| [Teams app capabilities](../concepts/design/map-use-cases.md#app-capabilities-mapped-to-features)| The Teams app capabilities are the core functionalities that you can build in your app. They're also called entry or extension points because they enable integration and interaction. <br> **See also**: [App scope](#a)|
```

Comment:
Maybe add something about these being defined in app manifest (to distinguish from TeamsJS capabilities below). For example "that you can build in your app, as defined in your app manifest."

Reply:
Updated as suggested